README.md

```diff
@@ -113,6 +113,10 @@ More learning resources can be found at:
 
 + https://github.com/mozilla/pdf.js/wiki/Additional-Learning-Resources
 
+### API reference
```

Comment:
Isn't this one level heading too deep, and also "reference" seems like a strange choice of word here...
Also, I'm not sure why the text from https://mozilla.github.io/pdf.js/api/ needs to be copied verbatim below either (since it adds even more links that may become stale).

Why not simply append the information to the list just above, e.g. something like this instead:
```
The API documentation can be found at:
+ https://mozilla.github.io/pdf.js/api/
```

Reply:
Added a commit with your suggested change, it fits much more nicely with the surrounding text.